sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -822,6 +822,7 @@ object FunctionRegistry {
 
     // Variant
     expression[ParseJson]("parse_json"),
+    expression[IsVariantNull]("is_variant_null"),
```

Comment:
I guess you need to look at `ExpressionsSchemaSuite`. It will fail if you only register here but haven't modified `sql-expression-schema.md`.

Reply:
Does regenerating golden files fix this issue?

Reply:
Richard confirmed to me that it does. I have done so. Resolving.